eng/common/TestResources/New-TestResources.ps1

```diff
@@ -393,6 +393,17 @@ foreach ($templateFile in $templateFiles) {
                 $DebugPreference = "Continue"
             }
             New-AzResourceGroupDeployment -Name $BaseName -ResourceGroupName $resourceGroup.ResourceGroupName -TemplateFile $templateFile -TemplateParameterObject $templateFileParameters
+        } catch {
+            $msg = @(
+                "",
+                "==================================================",
+                "For help debugging live test provisioning issues,",
+                "see http://aka.ms/azure-sdk-live-test-help",
+                "=================================================="
+            ) -join ([Environment]::NewLine)
+            Write-Output $msg
+
+            throw $_
```

Comment:
```suggestion
            throw
```
This should cause it to rethrow the same exception without changing things like the stack trace and stuff.